### DISCUSSION TITLE
Welcome to OpenCBM Discussions!

### DISCUSSION THREAD

**Discussion by spiro-trikaliotis** [OpenCBM/OpenCBM#84] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oe7twj** [comment]
My name is Tommy (Diddl at F64).

OpenCBM is great, I love it!

Currently I'm busy with MeGALoDOS.
This is a board for a 1541 drive with 1MB SRAM and 512KB Flash memory.
MeGALoDOS supports nearly all of this great hardware speeder of the past.

I did some changes on OpenCBM and made new tools supporting MeGALoDOS.
This changes are not complete, the work is in progress.

best regards

**Comment by ghost** [comment]
Hi, my name is kupababra. Greetings to everyone and the author of OpenCBM. 

I'm writing here because my floppy drive isn't working properly, but it will work! :)

I posted about the 1541 II, which I'm struggling with. 
I'm a fan of retro technology, I like to combine old with new, because not everything new is good ;)
